ci-operator/step-registry/acm-observability/tests/acm-observability-tests-ref.yaml

```diff
@@ -0,0 +1,18 @@
+ref:
+  as: acm-observability-tests
+  from: managed-services-integration-tests
+  commands: acm-observability-tests-commands.sh
+  resources:
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  env:
+    - name: HUB_CLUSTER_NAME
+      default: "ci-acm-hub"
+      documentation: Name of the hub cluster to run ACM Observability tests against
+    - name: TEST_MARKER
+      default: "acm_observability"
+      documentation: Test marker for running ACM Observability testsuite
+  documentation: >-
```

Comment:
please add that the cluster should be installed using `cluster-install` step registry

Reply:
done